src/components/MovieList.jsx

```diff
@@ -0,0 +1,119 @@
+import { useNavigate, useSearchParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import styles from "../styling/MovieList.module.css";
+import Category from "./Category";
+import Pagination from "./Pagination";
+import Lottie from "lottie-react";
+import loading from "../assets/loading.json";
+import LoadImage from "./LoadImage";
+import DataNotFound from "./DataNotFound";
+
+const Access_Token = import.meta.env.VITE_OPENDB_TOKEN;
+
+const Base_URL = "https://api.themoviedb.org/3/movie/";
+
+const MovieList = () => {
+  const navigate = useNavigate();
+  const [movies, setMovies] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState("");
+
+  const [searchParams, setSearchParams] = useSearchParams({
+    category: "now_playing",
+    page: 1,
+  });
+
+  useEffect(() => {
+    setMovies(null);
+    setIsLoading(true);
+    setError("");
+    const options = {
+      method: "GET",
+      headers: {
+        accept: "application/json",
+        Authorization: `Bearer ${Access_Token}`,
+      },
+    };
+    fetch(
+      `${Base_URL}${searchParams.get(
+        "category"
+      )}?language=en-US&page=${searchParams.get("page")}`,
+      options
+    )
+      .then(res => {
+        if (!res.ok) {
+          throw new Error("MovieList is unavailable.");
+        }
+        return res.json();
+      })
+      .then(res => {
+        setMovies(res.results);
+      })
+      .catch(err => setError(err.message))
+      .finally(() => setIsLoading(false));
+  }, [searchParams]);
+
+  const changePage = e => {
```

Comment:
We would have liked to implement it too if we had time. Nice solution! 💯 
//Martin, Linda